movie-app/src/containers/list-header-container/ListHeaderWrapper.tsx

```diff
@@ -0,0 +1,41 @@
+import styled from "styled-components";
+
+const ListHeaderWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  padding: 0 9.5%;
+  color: white;
+  background: #555555;
+
+  div {
```

Comment:
[optional] Why not class here?
Tag selectors are slower than class, as browser processes selector from their end. It means, that at first it will find all divs on a page, and only then filter them with parent class. If you need to reuse same style within all inner divs, you can create `styled.css` and extend form it

Reply:
Fixed